settings.gradle

```diff
@@ -1,6 +1,14 @@
 pluginManagement {
     plugins {
         id 'io.github.gradle-nexus.publish-plugin' version '1.1.0' apply false
+        id 'com.jfrog.artifactory' version settings['jfrog-artifactory.version']
```

Comment:
Consider use of the latest version of https://github.com/jfrog/artifactory-gradle-plugin + best practices from manual